packages/studio-be/src/core/migration/migration-service.ts

```diff
@@ -159,18 +113,30 @@ export interface MigrationOpts {
   configProvider: ConfigProvider
   database: Database
   inversify: Container
-  metadata: sdk.MigrationMetadata
+  metadata: MigrationMetadata
 }
 
 export type MigrationType = 'database' | 'config' | 'content'
```

Comment:
Difference between database, config and content?

Reply:
Absolutely none, it's just to display the type of change. Also forces us to split our changes in different migrations if we touch separate parts

Reply:
I understand that I mean when would you put database instead of config or content and same for config vs content?

Reply:
On the studio it should never be "database", but I left it there because it's the same "system" as the main repo